src/Uno.UI.Runtime.Skia.Tizen/Tizen/TizenHost.cs

```diff
@@ -0,0 +1,90 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using ElmSharp;
+using SkiaSharp;
+using SkiaSharp.Views.Tizen;
+using Tizen.NUI;
+using Uno.Foundation.Extensibility;
+using Windows.Graphics.Display;
+using Windows.UI.Xaml;
+using WinUI = Windows.UI.Xaml;
+using WUX = Windows.UI.Xaml;
+using TizenWindow = ElmSharp.Window;
+using Tizen.Applications;
+using Uno.UI.Runtime.Skia.Tizen;
+
+namespace Uno.UI.Runtime.Skia
+{
+	public class TizenHost : ISkiaHost
+	{
+		[ThreadStatic]
+		private static TizenHost _current;
+
+		private readonly TizenApplication _tizenApplication;
+		private readonly Func<WinUI.Application> _appBuilder;
+		private readonly TizenWindow _window;
+		private readonly string[] _args;
+
+		public static TizenHost Current => _current;
+
+		/// <summary>
+		/// Creates a WpfHost element to host a Uno-Skia into a WPF application.
+		/// </summary>
+		/// <remarks>
+		/// If args are omitted, those from Environment.GetCommandLineArgs() will be used.
+		/// </remarks>
+		public TizenHost(Func<WinUI.Application> appBuilder, string[] args = null)
+		{
+			_current = this;
+
+			_appBuilder = appBuilder;
+			_args = args;
+
+
+			_args ??= Environment
+				.GetCommandLineArgs()
+				.Skip(1)
+				.ToArray();
+			
+			Windows.UI.Core.CoreDispatcher.DispatchOverride
+				= (d) =>
+				{
+					//if (EcoreMainloop.IsMainThread)
+					//{
+					//	d();
+					//}
+					//else
+					//{
+						EcoreMainloop.PostAndWakeUp(d);
+					//}
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5993935)